content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +110,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up VM machines
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Install the Debian package with the Envoy sidecar:
+<!-- TODO: find a better way to reference release in the URL instead of manually sync... -->
+
+1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM. For example
 
     {{< text bash >}}
-    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
+    $ export GCE_NAME="your-gce-instance"
+    $ gcloud compute scp {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
     {{< /text >}}
 
-1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM.
+1.  Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ gcloud compute ssh "${GCE_NAME}"
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.6/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
 
 1.  Add the IP address of the Istio gateway (which we found in the [previous section](#preparing-the-kubernetes-cluster-for-expansion))
-    to `/etc/hosts` or to
-    the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is
-    an example of updating an `/etc/hosts` file with the Istio gateway address:
+to `/etc/hosts` or to
+the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is
+an example of updating an `/etc/hosts` file with the Istio gateway address:
 
     {{< text bash >}}
-    $ echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
+    $ sudo echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
     {{< /text >}}
 
 1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
 
     {{< text bash >}}
-    $ sudo mkdir /etc/certs
+    $ sudo mkdir -p /etc/certs
```

Comment:
This change is not needed since `/etc/` will already exist.

Reply:
I use `-p` this won't produce error.

We don't know what image and linux ditro customer is using, and this doesn't hurt.